OpenRA.Mods.Common/WebServices.cs

```diff
@@ -40,20 +40,24 @@ public void CheckModVersion()
 					{ "version", Game.ModData.Manifest.Metadata.Version }
 				}.ToString();
 
-				var client = HttpClientFactory.Create();
+				try
+				{
+					var client = HttpClientFactory.Create();
 
-				var httpResponseMessage = await client.GetAsync(queryURL);
-				var result = await httpResponseMessage.Content.ReadAsStringAsync();
+					var httpResponseMessage = await client.GetAsync(queryURL);
+					var result = await httpResponseMessage.Content.ReadAsStringAsync();
 
-				var status = ModVersionStatus.Latest;
-				switch (result)
-				{
-					case "outdated": status = ModVersionStatus.Outdated; break;
-					case "unknown": status = ModVersionStatus.Unknown; break;
-					case "playtest": status = ModVersionStatus.PlaytestAvailable; break;
-				}
+					var status = ModVersionStatus.Latest;
+					switch (result)
+					{
+						case "outdated": status = ModVersionStatus.Outdated; break;
+						case "unknown": status = ModVersionStatus.Unknown; break;
+						case "playtest": status = ModVersionStatus.PlaytestAvailable; break;
+					}
 
-				Game.RunAfterTick(() => ModVersionStatus = status);
+					Game.RunAfterTick(() => ModVersionStatus = status);
+				}
+				catch { }
```

Comment:
Worth a `Log.Write`?

Reply:
I don't think so - lack of an internet connection isn't *that* unusual (considering that this code is triggered from the main menu on launch), and we have a couple of other cases where failed queries don't log.